tutorials/surrogate/static_surrogate.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+from propulate import Surrogate
+from propulate.population import Individual
+
+
+class StaticSurrogate(Surrogate):
+    """
+    Surrogate model using the best known run as baseline.
+    After the first run, each subsequent loss is compared to the baseline.
+    Every run with a loss outside the margin of the baseline is cancelled.
+
+    This model assumes regular yields between training runs,
```

Comment:
this comment I don't understand

Reply:
It should mean that the yield in `ind_loss` has to be called regularly and without any randomness.
This is also mentioned in the surrogate class comment itself, but here again because in this particular instance it would break the static surrogate model directly.

I rewrote the comment in [7baed8](https://github.com/Helmholtz-AI-Energy/propulate/pull/112/commits/7baed87589cbaad58d7ec6dac318f3230e52fe73), which should be a little clearer.